plugins/MauticFocusBundle/Entity/StatRepository.php

```diff
@@ -39,4 +42,53 @@ public function getStats($id, $type, $fromDate = null)
 
         return $q->getQuery()->getArrayResult();
     }
+
+    /**
+     * @param array<string, mixed> $options
+     *
+     * @return array<string, mixed>
+     */
+    public function getStatsViewByLead(int $leadId, array $options = []): array
+    {
+        return $this->getStatsByLeadAndType($leadId, Stat::TYPE_NOTIFICATION, $options);
+    }
+
+    /**
+     * @param array<string, mixed> $options
+     *
+     * @return array<string, mixed>
+     */
+    public function getStatsClickByLead(int $leadId, array $options = []): array
+    {
+        return $this->getStatsByLeadAndType($leadId, Stat::TYPE_CLICK, $options);
+    }
+
+    /**
+     * @param array<string, mixed> $options
+     *
+     * @return array<string, mixed>
+     */
+    private function getStatsByLeadAndType(int $leadId, string $type, array $options = []): array
+    {
+        $q = $this->getEntityManager()->getConnection()->createQueryBuilder();
+
+        $q->from(MAUTIC_TABLE_PREFIX.'focus_stats', 's')
+            ->select('s.id, s.lead_id, s.type, s.date_added, f.id as focus_id, f.name as focus_name')
+            ->leftJoin('s', MAUTIC_TABLE_PREFIX.'focus', 'f', 's.focus_id=f.id');
+
+        $q->where($q->expr()->andX(
+            $q->expr()->eq('s.lead_id', (int) $leadId),
+            $q->expr()->like('s.type', '"'.$type.'"')
```

Comment:
I thought you wrote that you've changed the `like` into `eq` but it's still there. [I've read](https://stackoverflow.com/questions/543580/equals-vs-like) that `like` is 300 times slower compared to `eq`. It will differ based on the amount of data in the table, but still. Why not make it faster?
```suggestion
            $q->expr()->eq('s.type', ':type');
```
And somewhere bellow you'll set the param:
```php
$q->setParameter('type', $type);
```

Reply:
I changed this.